<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Logger</class>
 <widget class="QWidget" name="Logger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Log</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTableWidget" name="tableWidget">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string/>
      </property>
      <property name="toolTip">
       <string>Type of message</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Timestamp</string>
      </property>
      <property name="toolTip">
       <string>Original event timestamp</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Module</string>
      </property>
      <property name="toolTip">
       <string>Entity from which the message came from.</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Message</string>
      </property>
      <property name="toolTip">
       <string>The content of the message.</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonClear">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOkay">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Okay</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
